django_model_views/store/admin.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib import admin
+
+from .models import Category, Product
+
+
+@admin.register(Category)
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ['name', 'slug']
+    prepopulated_fields = {'slug': ('name',)}
+
+
+@admin.register(Product)
+class ProductAdmin(admin.ModelAdmin):
+    list_display = ['title', 'author', 'slug', 'price',
+                    'in_stock', 'created', 'updated']
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider showing is_active in list display</b></div>
<div id="fix">

Consider adding `is_active` to the `list_display` since it's used in `list_filter`. This would help admins quickly see and filter active/inactive products.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
    list_display = ['title', 'author', 'slug', 'price',
                    'in_stock', 'is_active', 'created', 'updated']
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/14#issuecomment-2533602906>#11cec0</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged